qa/restart-upgrade/src/test/java/org/opensearch/neuralsearch/bwc/AbstractRestartUpgradeRestTestCase.java

```diff
@@ -99,4 +101,11 @@ protected void createPipelineForSparseEncodingProcessor(final String modelId, fi
         );
         createPipelineProcessor(requestBody, pipelineName, modelId);
     }
+
+    protected void createPipelineForTextChunkingProcessor(String pipelineName) throws Exception {
```

Comment:
Can you make this identical to methods written above? Apart from this look good to me.

Reply:
The `.toURI()` operation in previous methods may introduce null pointer exception. I will implement a private function to read the file.

Reply:
Updated